SimTracker/Common/interface/SimHitInfoForLinks.h

```diff
@@ -8,10 +8,20 @@
 // Contains only the information needed to be make DigiSimLinks.
 // Include the simHit's index in the source collection, collection name suffix index.
 
-  struct SimHitInfoForLinks {
+  class SimHitInfoForLinks {
+  public:
     explicit SimHitInfoForLinks(PSimHit const* hitp, size_t hitindex, unsigned int tofbin) :
       eventId_(hitp->eventId()), trackIds_(1, hitp->trackId()), hitIndex_(hitindex), tofBin_(tofbin) {
     }
+
+    const EncodedEventId& eventId() const { return eventId_; }
+    const std::vector<unsigned int>& trackIds() const { return trackIds_; }
+    std::vector<unsigned int>& trackIds() { return trackIds_; } // needed ATM in phase2 digitizer
+    unsigned int trackId() const { return trackIds_[0]; }
+    size_t hitIndex() const { return hitIndex_; }
+    unsigned int tofBin() const { return tofBin_; }
+
+  private:
     EncodedEventId eventId_;
     std::vector<unsigned int> trackIds_;
```

Comment:
If this PR is the way to go, eventually `trackIds_` should be changed to plain `unsigned int`. I left it as `vector` to avoid touching phase2 digitizer (for now).